library/pycount/src/pycount/characters.py

```diff
@@ -8,3 +8,6 @@ def count_characters(string):
         counts[character] += 1
 
     return dict(counts)
+
+def count_characters_ignoreCase(string):
+    return count_characters(string.upper())
```

Comment:
S.v.p. corriger:

library/pycount/src/pycount/characters.py:12:1: E302 expected 2 blank lines, found 1
library/pycount/src/pycount/characters.py:13:44: W292 no newline at end of file